tools/cldr-code/src/main/java/org/unicode/cldr/tool/ExtractMessages.java

```diff
@@ -205,12 +206,7 @@ public void handlePathValue(String path, String value) {
         }
     }
 
-    public static Collator col = Collator.getInstance(ULocale.ROOT); // freeze below
-
-    static {
-        col.setStrength(Collator.SECONDARY);
-        col = col.freeze();
-    }
+    public static Collator col = CollatorHelper.ROOT_SECONDARY;
```

Comment:
Mark, you pointed out this one when reviewing the previous PR

Reply:
ah